### ISSUE TITLE
Accessibility issues with floating labels/validation

### ISSUE THREAD

**Issue by FurcoFucho** [miguelDeDiego/GitHub-Tutorial#2]
We are using the floating label look (https://getbootstrap.com/docs/5.1/forms/floating-labels/) for our forms and we've run into an accessibility issue where in High Contrast mode, users expected that the fields and text inside should be in proper orientation since this may cause confusion for our keyboard users
